R/SerializeUtils.R

```diff
@@ -0,0 +1,69 @@
+.loadJson <- function(definition, simplifyVector = FALSE, simplifyDataFrame = FALSE, ...) {
```

Comment:
These functions are carry-overs from CohortIncidence.  I did a lot of testing in CI for handling serialization and I wanted to carry those learnings forward to here.